src/container/RelayContainer.js

```diff
@@ -106,6 +107,7 @@ function createContainerComponent(
   var fragmentNames = Object.keys(fragments);
   var initialVariables = spec.initialVariables || {};
   var prepareVariables = spec.prepareVariables;
+  var pure = spec.pure !== false;
```

Comment:
This is written as a double negative to handle the case when `spec.pure` is `undefined` and it should default to `true`.